src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,14 @@
+.star {
+  background-image: url("../../src/images/star-active.svg");
```

Comment:
You need to rewrite this logic. There should be a number of active stars in the wrapper class, and styles should hang on the required elements based on this number, for this use `nth-child`